modules/siddhi-extensions/map/src/test/java/org/wso2/siddhi/extension/map/PutFunctionExtensionTestCase.java

```diff
@@ -145,4 +146,43 @@ public void receive(Event[] events) {
         Assert.assertTrue(eventArrived);
         executionPlanRuntime.shutdown();
     }
+
+    @Test(expected = ExecutionPlanValidationException.class)
+    public void testPutFunctionExtension1() throws InterruptedException {
+        log.info("PutFunctionExtension TestCase with test attributeExpressionExecutors length");
+        SiddhiManager siddhiManager = new SiddhiManager();
+
+        String inStreamDefinition = "\ndefine stream inputStream (symbol string, price long, volume long);";
+        String query = ("@info(name = 'query1') from inputStream select symbol,price, "
+                + "map:create() as tmpMap insert into tmpStream;"
+                + "@info(name = 'query2') from tmpStream  select symbol,price,tmpMap, map:put(tmpMap,symbol) as newmap"
+                + " insert into outputStream;"
+
+        );
+
+        ExecutionPlanRuntime executionPlanRuntime = siddhiManager.createExecutionPlanRuntime(inStreamDefinition + query);
```

Comment:
Below three lines are not needed. createExecutionplanRuntime will throw the exception